CameraInspector/WebCameraCapture.cpp

```diff
@@ -0,0 +1,19 @@
+#include "WebCameraCapture.h"
+
+namespace CameraInspector
+{
+
+void WebCameraCapture::Start()
+{
+}
+
+void WebCameraCapture::Stop()
+{
+}
+
+void WebCameraCapture::AddFrameHandler(std::shared_ptr<IFrameHandler>& handler)
+{
+	handlers_.push_back(std::move(handler));
```

Comment:
Implicit behavior: I passed to your function my handler and after that I can't access to my handler. Why?
Use std::weak_ptr or std::unique_ptr for such functionality.